packages/ui/src/accounts/modals/MoveFoundsModal/MoveFoundsTransferableModal.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+
+import { AccountInfo } from '@/accounts/components/AccountInfo'
+import { AddressToBalanceMap, Account } from '@/accounts/types'
+import { ButtonPrimary } from '@/common/components/buttons'
+import { HelpIcon } from '@/common/components/icons'
+import { Modal, ModalHeader, BalanceInfoInRow, InfoTitle, InfoValue, ModalFooter } from '@/common/components/Modal'
+import { TextMedium, TokenValue } from '@/common/components/typography'
+import { Colors } from '@/common/constants'
+import { spacing } from '@/common/utils/styles'
+
+import { ModalBody, MemberRow } from './styles'
+
+interface Props {
+  onClose: () => void
+  onManageAccountsClick: () => void
+  price: number
```

Comment:
This should be named: `requiredStake` as the name reflects the domain naming. See below :point\_down: . Some of the actions invoked by the users require [staking](https://joystream.gitbook.io/joystream-handbook/key-concepts/staking). If the user has no account with a required stake then the app should detect if there's is any combination of the accounts' balances that allows gathering required stake.